help.go

```diff
@@ -89,9 +89,6 @@ func DefaultHelpValueFormatter(value *Value) string {
 
 // DefaultHelpPrinter is the default HelpPrinter.
 func DefaultHelpPrinter(options HelpOptions, ctx *Context) error {
-	if ctx.Empty() {
-		options.Summary = false
-	}
```

Comment:
Not sure if this is OK.

Reply:
Why remove it, was it doing something unexpected?

Reply:
The usage message doesn't get printed as short (but long/full) is I keep these lines.
`helopoOtions.Summary` is how indicate that I want the short usage - maybe that was a bad choice?
 `options.Summary = true` is set in under `ShortUsageOnError` in `kong.go` and then reset here.